confgenerator/fluentbit/inputs.go

```diff
@@ -92,7 +92,15 @@ func (i Syslog) Component() Component {
 			// as a hint to set "how much data can be up in memory", once the limit is reached it continues writing to disk.
 			"Mem_Buf_Limit": "10M",
 		},
-	}
+	}, {
+		// FIXME: This is not new, but we shouldn't be disabling syslog protocol parsing by passing a custom Parser - Fluentbit includes builtin syslog protocol support, and we should enable/expose that.
```

Comment:
Agree that we can add a config parameter to control the syslog format. The default would have to stay raw though to avoid breaking changes.